### PR TITLE
Wb pie chart

### PR DESCRIPTION
**Pull request by WillBanks1**
- Created two-layer pie chart for PDF report
- Added logging to handler function

### CODE REVIEW

File: dashboard/main.tf [hunk 1]
```diff
@@ -0,0 +1,112 @@
+provider "aws" {
+    region = var.AWS_REGION
+    access_key = var.AWS_ACCESS_KEY
+    secret_key = var.AWS_SECRET_KEY
+  
+}
+
+# 1 Dashboard Task Definition
```
Comment (fm1psy) [inline dashboard/main.tf:8]: Good use of comments to clarify the purpose of each object, I like it

File: weekly-report/main.py [hunk 2]
```diff
@@ -115,20 +121,21 @@ def handler(event=None, context=None):
...
     file_name = f"{CURRENT_DATE}.pdf"
     upload_historical_readings(
         s_client, ENV['STORAGE_BUCKET_NAME'], prefix, file_name, file_data)
-
-    return {"success": "complete"}
+    logging.info(f"Weekly PDF report uploaded to {ENV['STORAGE_BUCKET_NAME']}")
 
 
 if __name__ == "__main__":
+    logging.basicConfig(level=logging.INFO)
```
Comment (fm1psy) [inline weekly-report/main.py:140]: I like the use of logging to create messages that can be viewed on the log system in aws

**Review by fm1psy — DISMISSED** [review]
It looks ready to merge, good job

**Review by fm1psy — APPROVED** [review]
This looks ready to merge, thanks

**Review by joe1606 — APPROVED** [review]
like the pie chart idea

**Review by joe1606 — APPROVED** [review]
looks good

**Pull request closed without merging**